library/modules/form/controls/scss/checkbox.scss

```diff
@@ -1,5 +1,5 @@
 .pragmate-checkbox {
```

Comment:
the css class must be named "pui" following the standard defined @FranciscoSanvicente